src/test/mod.rs

```diff
@@ -172,6 +172,11 @@ fn system_tests() {
         // Display results.
         println!("Ran {} system tests.", count);
         assert_eq!(fails, 0, "{} system tests failed", fails);
+        assert!(
+            count >= 300,
+            "Expected a minimum of {} system tests to be executed",
+            300
+        )
```

Comment:
I started out with this just making sure that `count` was greater than zero, but given the nature of this check I thought it would be more helpful to use a higher floor. If the number of tests drops dramatically (for example from the 378 system tests currently to < 300 tests) then I think that would _likely_ be an indication of something unintentional/accidental (like a skip pattern that's too broad).

Let me know if folks think this higher floor will make this test too brittle though and I'll be happy to change it.

Reply:
Seems a good idea to me :+1: